README.md

```diff
@@ -234,6 +234,7 @@ Team members are listed in alphabetical order.
 * [__Vincent Le Goff__](https://github.com/zekth)
 * [__Maksim Sinik__](https://github.com/fox1t), <https://twitter.com/maksimsinik>, <https://www.npmjs.com/~fox1t>
 * [__Manuel Spigolon__](https://github.com/eomm), <https://twitter.com/manueomm>, <https://www.npmjs.com/~eomm>
+* [__Salman Mitha__](https://github.com/salmanm), <https://www.npmjs.com/~salmanm>
```

Comment:
This line should be before line 235.

Reply:
Alphabetical order

Reply:
Ah, by **last** name. Didn't notice. 🤦
thanks